stellarphot/gui_tools/comparison_functions.py

```diff
@@ -378,10 +380,22 @@ def _show_label_button_handler(self, change):
 
     def _save_aperture_to_file(self, button=None, filename=''):
         if not filename:
-            filename = self.aperture_output_file
+            filename = self.photom_apertures_file
         # Export aperture file as CSV (overwrite existing file if it exists)
         try:
-            self.generate_table().write(filename, overwrite=self.overwrite_outputs)
+            # Convert aperture table into a SourceList objects and output
```

Comment:
I think most of this (except the last `sources.write....`) should come before the `try` since we are only checking for the `OSError`.

Reply:
I see you already pushed this change through.